<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ControlDialog</class>
 <widget class="QDialog" name="ControlDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>584</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Analytics control</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <widget class="QLabel" name="descrLabel1">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>30</y>
         <width>441</width>
         <height>311</height>
        </rect>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Do you want to help us improve Qt Creator?&lt;br/&gt;&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;By enabling analytics telemetry gathering, Qt Creator usage data will be sent to us. The data will help us to improve Qt in the future. &lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;How is my privacy protected?&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;&lt;br/&gt;&lt;/span&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;Collected data contains no personal data and is not connected to you, your project, location or Qt Account.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="scaledContents">
        <bool>false</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>101</width>
         <height>91</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="plugin.qrc">:/images/monitoring-300px.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_11">
       <item>
        <widget class="QLabel" name="descrLabel2">
         <property name="font">
          <font>
           <italic>true</italic>
          </font>
         </property>
         <property name="text">
          <string>If enabled, the analytics can be disabled at any time.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Enable telemetry data gathering</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_1">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="telemetryLabel">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>Telemetry Data Sources</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_1">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QGroupBox" name="generalGroupBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>General</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_9">
               <item>
                <widget class="QCheckBox" name="applicationVersionCheckBox">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Data source for the application version. The application version is retrieved via &lt;span style=&quot; font-weight:600; color:#aa557f;&quot;&gt;QCoreApplication::applicationVersion.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="text">
                  <string>Application Version</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="compilerInfoCheckBox">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Data source reporting which compiler was used to build this code. &lt;/p&gt;&lt;blockquote&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Note&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-style:italic;&quot;&gt;This will report which compiler was used to build the feedback library, which technically does not need to be the same as used for the application code. This is particularly true for compilers that don't break ABI regularly (such as Clang and GCC), so this information is most reliable when this is not the case, e.g. with MSVC.&lt;/span&gt;&lt;/blockquote&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="text">
                  <string>Compiler Info</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="cpuInfoCheckBox">
                 <property name="toolTip">
                  <string>Data source reporting the type and amount of CPUs.</string>
                 </property>
                 <property name="text">
                  <string>CPU Info</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="localeInfoCheckBox">
                 <property name="toolTip">
                  <string>Data source reporting the region and language settings.</string>
                 </property>
                 <property name="text">
                  <string>Locale Info</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="openGLInfoCheckBox">
                 <property name="toolTip">
                  <string>Data source reporting type, version and vendor of the OpenGL stack.</string>
                 </property>
                 <property name="text">
                  <string>OpenGL Info</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="platformInfoCheckBox">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Reports information about the platform the application is running on. This includes two string data fields:&lt;/p&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;platform.os: the operating system name&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;platform.version: the operating system or distribution version.&lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="text">
                  <string>Platform Info</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="qPAInfoCheckBox">
                 <property name="toolTip">
                  <string>Reports information about the Qt platform abstraction plugin the application is running on.</string>
                 </property>
                 <property name="text">
                  <string>Qt platform abstraction plugin</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="qtVersionCheckBox">
                 <property name="toolTip">
                  <string>Data source reporting the Qt version used at runtime.</string>
                 </property>
                 <property name="text">
                  <string>Qt Version</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="screenInfoCheckBox">
                 <property name="toolTip">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Data source for information about connected displays. This provides as array of maps containing the following properties:&lt;/p&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;height: Height of the corresponding screen in pixel.&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;width: Width of the corresponding screen in pixel.&lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;dpi: Dots per inch of the corresponding screen.&lt;/li&gt;&lt;/ul&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                 <property name="text">
                  <string>Screen Info</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="startCountCheckBox">
                 <property name="toolTip">
                  <string>Data source reporting the total amount of applications starts.</string>
                 </property>
                 <property name="text">
                  <string>Start Count</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="styleInfoCheckBox">
                 <property name="toolTip">
                  <string>Data source the widget style and color scheme used by the application. </string>
                 </property>
                 <property name="text">
                  <string>Style Info</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="usageTimeCheckBox">
                 <property name="toolTip">
                  <string>Data source reporting the total usage time of the application.</string>
                 </property>
                 <property name="text">
                  <string>Usage Time</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="creatorGroupBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>1</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="title">
               <string>Qt Creator specific</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_10">
               <item>
                <widget class="QListView" name="listView"/>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>24</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="helpButton">
        <property name="text">
         <string>&amp;Help</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>198</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="telemetryButton">
        <property name="text">
         <string> &amp;Enable telemetry ... </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="okButton">
        <property name="text">
         <string>&amp;Ok</string>
        </property>
        <property name="icon">
         <iconset theme="SP_DialogCancelButton"/>
        </property>
        <property name="default">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelButton">
        <property name="text">
         <string>&amp;Cancel</string>
        </property>
        <property name="icon">
         <iconset theme="Ok"/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="plugin.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>okButton</sender>
   <signal>clicked()</signal>
   <receiver>ControlDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ControlDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>388</y>
    </hint>
    <hint type="destinationlabel">
     <x>291</x>
     <y>202</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
